src/ejabberd_oauth.erl

```diff
@@ -547,7 +547,7 @@ process(_Handlers,
                     ?C(<<" is maintained by ">>),
                     ?XAC(<<"a">>,
                          [{<<"href">>, <<"https://www.process-one.net">>},
-                          {<<"title">>, <<"ProcessOne - Leader in Instant Messaging and Push Solutions">>}],
+                          {<<"title">>, <<"ProcessOne—Leader in Instant Messaging and Push Solutions">>}],
```

Comment:
```suggestion
                          {<<"title">>, <<"ProcessOne—Instant Messaging and Push Solutions">>}],
```

Reply:
Your next step should be something like
```diff
- 2001 - A Space Oddyssey
+ 2001 - A Guy Takes a Peaceful Walk on Sky
```

Reply:
I have discovered the problem here.
Better to revert this part.

Reply:
The problem being?

Reply:
Original:
- "ProcessOne - Leader in Instant Messaging and Push Solutions"

Your version:
- "ProcessOne—Leader in Instant Messaging and Push Solutions"

Reply:
That is an em dash. I meant to do that, if it makes a difference.